package.json

```diff
@@ -84,14 +89,23 @@
       "stylelint-no-unsupported-browser-features"
     ],
     "rules": {
+      "string-quotes": "single",
+      "color-hex-length": "long",
+      "function-parentheses-space-inside": "always",
```

Comment:
I wouldn't enforce this as a rule. We can recommend that, but the linter should report more important issues I believe. Other than for consistency reasons, it doesn't make code that much easier to read (as opposed to what happens in other languages where you might have 5 parameters to a function).